components/rhel/rhel-ose/Vagrantfile

```diff
@@ -17,7 +23,7 @@ VM_MEMORY = ENV['VM_MEMORY'] || 3072
 #PROXY_PASSWORD = "mysecretpass"
```

Comment:
Not so fond how this is done, by commenting it out. Would it not be nice to align with how we do things with the subscription details? Why do we not env variables there?